org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/model/ListLineTracker.java

```diff
@@ -209,7 +209,7 @@ public final int getLineOffset(int line) throws BadLocationException {
 		int lines= fLines.size();
 
 		if (line < 0 || line > lines)
-			throw new BadLocationException();
+			throw new BadLocationException("The line value, {" + line +"}, is out of bounds.");

```

Comment:
Please commit that in a new PR (commit it yourself)

Reply:
Done